css/style.css

```diff
@@ -241,35 +241,36 @@ button:hover {
 	font-weight: 700;
 	overflow: hidden;
 	height: 40px;
+	display: flex;
 }
 
 .stack h2 input {
 	padding: 0;
-	width: 80%;
+	width: 100%;
 	margin: 0;
 	font-size: 12pt;
 	font-weight: 700;
 	border: 0;
 	background-color: transparent;
-	float: left;
 	min-height: initial;
 }
 
 .stack h2 button,
 .stack .stack-actions {
 	float: right;
+	margin: 3px 1px 3px 19px;
```

Comment:
@juliushaertl it looks very poor...

![screenshot_20170504_215812](https://cloud.githubusercontent.com/assets/16517095/25722515/d0a38212-3114-11e7-8a8b-6f4fac00f3bd.png)